packages/babel-plugin/src/__tests__/index.test.tsx

```diff
@@ -37,11 +37,11 @@ describe('babel plugin', () => {
 
     expect(output?.code).toMatchInlineSnapshot(`
       "import React from \\"react\\";
-      import { Style } from '@compiled/css-in-js';
+      import { CC, Style } from '@compiled/css-in-js';
```

Comment:
every component now wrapped with "CC" component - this will have context on server and noop on client (immediately return children)